### PR TITLE
[ETP-436] ET - My Tickets - Show IAC re-send email checkbox

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ETP-436]

🎫 [ETP-436]: https://moderntribe.atlassian.net/browse/ETP-436
🎥 https://www.loom.com/share/2d96ed3b6e0a449f858a408ea0e382cc

https://github.com/moderntribe/event-tickets-plus/pull/1063

### CODE REVIEW

File: src/views/tickets/orders.php [hunk 1]
```diff
@@ -105,45 +104,54 @@
...
+			?>
+
+			<?php
+			/**
+			 * Fires before the process tickets submission button is rendered
+			 */
+			do_action( 'tribe_tickets_orders_before_submit' );
+			?>
+
+			<?php if (
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:130]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: fe33760c8f100ef49a181f9914703494</sub>

File: src/views/tickets/orders.php [hunk 2]
```diff
@@ -105,45 +104,54 @@
...
+				// Current user has RSVP (with or without meta) so needs to be able to edit status
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:131]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 966024dd7cab18750ed4710073fc6001</sub>

File: src/views/tickets/orders.php [hunk 3]
```diff
@@ -105,45 +104,54 @@
...
+				$view->has_rsvp_attendees( $event_id, get_current_user_id() )
+				|| (
+					// Current user has tickets with meta so needs to be able to edit meta
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:134]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 17cf939c85dc381f8de2ea247811b116</sub>

File: src/views/tickets/orders.php [hunk 4]
```diff
@@ -105,45 +104,54 @@
...
+					$view->has_ticket_attendees( $event_id, get_current_user_id() )
+					&& $tribe_my_tickets_have_meta
+				)
+			) : ?>
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:138]: :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 288422e8a82522a1eba9b3231dbee181</sub>

File: src/views/tickets/orders.php [hunk 5]
```diff
@@ -105,45 +104,54 @@
...
+				<div class="tribe-submit-tickets-form">
+					<button
+						type="submit"
+						name="process-tickets"
+						value="1"
+						class="button alt"
+					>
+						<?php echo sprintf( esc_html__( 'Update %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, get_current_user_id() ) ); ?>
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:146]: * :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_current_user_id'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: fea64958e59f5f20b7f688a5b138189d</sub>

File: src/views/tickets/orders.php [hunk 6]
```diff
@@ -105,45 +104,54 @@
...
+					</button>
+				</div>
+			<?php endif;
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:149]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: c0bfc93528ebd6a1529be07ece586ebc</sub>

File: src/views/tickets/orders.php [hunk 7]
```diff
@@ -105,45 +104,54 @@
...
+			// unset our global since we don't need it any more
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:150]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: c47855c14f7f1f24aad7a41369efe462</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 9 error(s)
:warning: 2 warning(s)

File: src/views/tickets/orders.php [hunk 8]
```diff
@@ -105,45 +114,55 @@
...
+			?>
+
+			<?php
+			/**
+			 * Fires before the process tickets submission button is rendered
+			 */
+			do_action( 'tribe_tickets_orders_before_submit' );
+			?>
+
+			<?php if (
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:141]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 932ced5a17a51261a115cf637da45f3e</sub>

File: src/views/tickets/orders.php [hunk 9]
```diff
@@ -105,45 +114,55 @@
...
+				// Current user has RSVP (with or without meta) so needs to be able to edit status
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:142]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3aaddab568842cb1ac9b6f740146cd74</sub>

File: src/views/tickets/orders.php [hunk 10]
```diff
@@ -105,45 +114,55 @@
...
+				$view->has_rsvp_attendees( $event_id, get_current_user_id() )
+				|| (
+					// Current user has tickets with meta so needs to be able to edit meta
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:145]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ff585f099e3444da7135b9def547753a</sub>

File: src/views/tickets/orders.php [hunk 11]
```diff
@@ -105,45 +114,55 @@
...
+					$view->has_ticket_attendees( $event_id, get_current_user_id() )
+					&& $tribe_my_tickets_have_meta
+				)
+			) : ?>
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:149]: :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: aef16f25c2d654bf7d232beead76bffd</sub>

File: src/views/tickets/orders.php [hunk 12]
```diff
@@ -105,45 +114,55 @@
...
+				<div class="tribe-submit-tickets-form">
+					<button
+						type="submit"
+						name="process-tickets"
+						value="1"
+						class="button alt"
+					>
+						<?php echo sprintf( esc_html__( 'Update %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, get_current_user_id() ) ); ?>
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:157]: * :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_current_user_id'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 20889e13c5d791174486c682b8d31f5d</sub>

File: src/views/tickets/orders.php [hunk 13]
```diff
@@ -105,45 +114,55 @@
...
+					</button>
+				</div>
+			<?php endif;
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:160]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 1b23118f5fd621ba3052960431ad1df8</sub>

File: src/views/tickets/orders.php [hunk 14]
```diff
@@ -105,45 +114,55 @@
...
+			// unset our global since we don't need it any more
```
Comment (tr1b0t) [inline src/views/tickets/orders.php:161]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a0243db49e8b54799bd0d7f5ff875b54</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 9 error(s)
:warning: 2 warning(s)

File: src/views/tickets/orders.php (hunk 1 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:130]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: fe33760c8f100ef49a181f9914703494</sub>

File: src/views/tickets/orders.php (hunk 2 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:131]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 966024dd7cab18750ed4710073fc6001</sub>

File: src/views/tickets/orders.php (hunk 3 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:134]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 17cf939c85dc381f8de2ea247811b116</sub>

File: src/views/tickets/orders.php (hunk 4 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:138]: :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 288422e8a82522a1eba9b3231dbee181</sub>

File: src/views/tickets/orders.php (hunk 5 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:146]: * :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_current_user_id'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: fea64958e59f5f20b7f688a5b138189d</sub>

File: src/views/tickets/orders.php (hunk 6 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:149]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: c0bfc93528ebd6a1529be07ece586ebc</sub>

File: src/views/tickets/orders.php (hunk 7 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:150]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: c47855c14f7f1f24aad7a41369efe462</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 9 error(s)
:warning: 2 warning(s)

File: src/views/tickets/orders.php (hunk 8 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:141]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 932ced5a17a51261a115cf637da45f3e</sub>

File: src/views/tickets/orders.php (hunk 9 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:142]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3aaddab568842cb1ac9b6f740146cd74</sub>

File: src/views/tickets/orders.php (hunk 10 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:145]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ff585f099e3444da7135b9def547753a</sub>

File: src/views/tickets/orders.php (hunk 11 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:149]: :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: aef16f25c2d654bf7d232beead76bffd</sub>

File: src/views/tickets/orders.php (hunk 12 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:157]: * :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_current_user_id'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 20889e13c5d791174486c682b8d31f5d</sub>

File: src/views/tickets/orders.php (hunk 13 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:160]: :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 1b23118f5fd621ba3052960431ad1df8</sub>

File: src/views/tickets/orders.php (hunk 14 above)
Comment (tr1b0t) [inline src/views/tickets/orders.php:161]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a0243db49e8b54799bd0d7f5ff875b54</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 9 error(s)
:warning: 2 warning(s)

**Review by sc0ttkclark — COMMENTED** [review]
✅

**Review by bordoni — APPROVED** [review]

**Pull request merged**